lib/common/options.js

```diff
@@ -195,5 +195,8 @@ Options.from = function (_options) {
   options.build.transpile = [].concat(options.build.transpile || [])
     .map(module => module instanceof RegExp ? module : new RegExp(module))
 
+  // User-defined changes
+  options.watch = []
```

Comment:
I think this should move to https://github.com/nuxt/nuxt.js/blob/dev/lib/common/nuxt.config.js (I will use this file in the docs to display all configurations)